src/test/resources/com/tresys/nitf/xsd/nitf_ude_types.dfdl.xsd

```diff
@@ -0,0 +1,156 @@
+<?xml version='1.0' encoding='UTF-8'?>
+
+<!-- Copyright (c) 2014 Tresys Technology, LLC. All rights reserved. Developed by: 
+  Tresys Technology, LLC http://www.tresys.com Permission is hereby granted, free of 
+  charge, to any person obtaining a copy of this software and associated documentation 
+  files (the "Software"), to deal with the Software without restriction, including 
+  without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, 
+  and/or sell copies of the Software, and to permit persons to whom the Software is 
+  furnished to do so, subject to the following conditions: 1. Redistributions of source 
+  code must retain the above copyright notice, this list of conditions and the following 
+  disclaimers. 2. Redistributions in binary form must reproduce the above copyright 
+  notice, this list of conditions and the following disclaimers in the documentation 
+  and/or other materials provided with the distribution. 3. Neither the names of Tresys 
+  Technology, nor the names of its contributors may be used to endorse or promote products 
+  derived from this Software without specific prior written permission. THE SOFTWARE 
+  IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING 
+  BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE 
+  AND NONINFRINGEMENT. IN NO EVENT SHALL THE CONTRIBUTORS OR COPYRIGHT HOLDERS BE LIABLE 
+  FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT 
+  OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE 
+  OR OTHER DEALINGS WITH THE SOFTWARE. -->
+
+<xs:schema targetNamespace="urn:nitfUserDefinedExtensions" xmlns:tns="urn:nitfUserDefinedExtensions"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:fn="http://www.w3.org/2005/xpath-functions"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
+  xmlns:dfdlx="http://www.ogf.org/dfdl/dfdl-1.0/extensions"
+  xmlns:nitf="urn:nitf:2.1"
+  elementFormDefault="unqualified">
+
+  <xs:import namespace="urn:nitf:2.1" schemaLocation="com/tresys/nitf/xsd/nitf.dfdl.xsd" />
```

Comment:
Is this import needed to use the simple types defined in nitf? Maybe we should split those types into a nitf-types-types.dfld.xsd or something like that? That way there's a clear separation between the main nitf schema and the types that extensions can use?

Reply:
I actually had done that earlier when trying to get past the class path issues.  I think it makes sense to make the change and I like the idea of splitting up the NITF project into 2 separate projects.